### DISCUSSION TITLE
Clicks in DeepEdit Inference

### DISCUSSION THREAD

**Discussion by annkapopp** [Project-MONAI/MONAILabel#1592] [Q&A]
Hey,

I was wondering if the clicks are included in the (batch) inference function on the server for DeepEdit or if it is just the automatic segmentation? If the clicks are included, where can I set the deepgrow_probability?

Thank you!

**Comment by diazandr3s** [comment]
Hi @annkapopp,

For inference, DeepEdit does not use deepgrow probability - you either run DeepEdit in the automatic inference mode or the user provides the clicks.

If you mean for training, here is that argument: https://github.com/Project-MONAI/MONAILabel/blob/ac0af6d4886bcd10f4a7f09419446ba0ba20eca0/sample-apps/radiology/lib/trainers/deepedit.py#L59

Hope this helps,
